From ae222e12c8a78c6154dcebab158900e67b53a11d Mon Sep 17 00:00:00 2001
From: Stephen L Arnold <sarnold@vctlabs.com>
Date: Sat, 8 Jul 2023 19:33:21 -0700
Subject: [PATCH 3/5] remove 3 conflicting checks from standard profile

* systemd config does not exist, backup perms are one step too far

Signed-off-by: Stephen L Arnold <sarnold@vctlabs.com>
---
 products/openembedded/profiles/standard.profile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/products/openembedded/profiles/standard.profile b/products/openembedded/profiles/standard.profile
index 72aac10c02..2dac68d8ef 100644
--- a/products/openembedded/profiles/standard.profile
+++ b/products/openembedded/profiles/standard.profile
@@ -129,8 +129,6 @@ selections:
     - require_singleuser_auth
     - require_emergency_target_auth
     - disable_users_coredumps
-    - coredump_disable_backtraces
-    - coredump_disable_storage
     - configure_crypto_policy
     - var_system_crypto_policy=default_policy
     - dir_perms_world_writable_sticky_bits
@@ -153,7 +151,6 @@ selections:
     - file_permissions_backup_etc_group
     - file_groupowner_backup_etc_gshadow
     - file_owner_backup_etc_gshadow
-    - file_permissions_backup_etc_gshadow
     - file_permissions_unauthorized_world_writable
     - file_permissions_ungroupowned
     - accounts_root_path_dirs_no_write
-- 
2.43.0

